### ISSUE TITLE
Carthage: Parse error: expected submodule commit SHA in output of task (ls-tree -z v1.2.1 Carthage/Checkouts/Runes) but encountered: 

### ISSUE THREAD

**Issue by evgeniyd** [kaishin/Gifu#71]
While using Carthage to install the library, I got the output:

    $ carthage update --verbose
    *** Fetching Gifu
    *** Checking out Gifu at "v1.2.1"
    Parse error: expected submodule commit SHA in output of task (ls-tree -z v1.2.1 Carthage/Checkouts/Runes) but encountered: 

I fixed this issue by cleaning up `/Carthage/Checkouts/` and running `$ carthage update --use-submodules`

Probably the README has to be updated.

Carthage version is **0.17.2**

**Comment by evgeniyd** [comment]
It is fixed in [72e8a5ea469f479f4cd3089eb0208a3588823b45] 

**Note:** I didn't submitted it with PR #73 yet

**Comment by evgeniyd** [comment]
**NOTE:**

> I fixed this issue by cleaning up /Carthage/Checkouts/ and running `$ carthage update --use-submodules`

this is not an appropriate fix, b/c it changes your project's git settings to use submodules, which in my case is not good at all.

Which is why the fix in the PR works for me.

**Comment by Quarezz** [comment]
Encountered this issue today:
```
$ carthage update

*** Checking out Gifu at "v1.2.1"
Parse error: expected submodule commit SHA in output of task (ls-tree -z v1.2.1 Carthage/Checkouts/Runes) but encountered:
```

**Comment by evgeniyd** [comment]
@Quarezz did you use [my fork](https://github.com/evgeniyd/Gifu/tree/evgeniyd-carthage-submodule-fix)?

What I see from your logs, is exactly what I used to see until the fix.

**Comment by Quarezz** [comment]
Moved to using UIImageView.animationImages so didn't check, thanks

**Comment by kaishin** [comment]
Fixed in #74.
